.circleci/config.yml

```diff
@@ -17,6 +17,8 @@ jobs:
       RAILS_ENV: test
       NODE_ENV: test
       RACK_ENV: test
+      COVERAGE: true
+      CC_TEST_REPORTER_ID: 04daa6564351115dc1515504790cd379ad8dc25e7778f0641e0f8c63185f887c
```

Comment:
Required in all builds based on [this issue thread](https://github.com/codeclimate/test-reporter/issues/413)